File: src/my_project/easy_problems/from251to300/count_elements_strictly_smaller_and_greater.py
from typing import List 

class Solution:
    def countElements(self, nums: List[int]) -> int:
        
        
        min_num: int = min(nums)
        max_num: int = max(nums)
        
        count = 0
        
        for num in nums:
            if num > min_num and num < max_num:
                count += 1
            
            
            
        
        return count
    
    
solution = Solution()

print(solution.countElements([11,7,2,15]))

print(solution.countElements([-3,3,3,90]))

File: src/my_project_typescript/easy_problems_typescript/from151to200/countElementsStrictlySmallerAndGreater.ts
function countElements(nums: number[]): number {

    let maxNum: number = nums.reduce((a,b) => Math.max(a,b))
    let minNum: number = nums.reduce((a,b) => Math.min(a,b))

    let answer: number = 0

    nums.map(a => a > minNum && a < maxNum ? answer += 1:0)

    return answer 

};